relay-server/src/services/cogs.rs

```diff
@@ -1,9 +1,10 @@
 use std::sync::atomic::{AtomicBool, Ordering};
 
+use sentry_usage_accountant::{Producer, UsageAccountant, UsageUnit};
```

Comment:
nit: We treat external dependencies the same as other relay crates for import order, so this was in the right place IMO.

Reply:
It's the automatic imports formatting that messes with this. Will fix